client/my-sites/themes/pattern-assembler-button.tsx

```diff
@@ -0,0 +1,18 @@
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import './pattern-assembler-button.scss';
+
+type Props = {
+	isPrimary?: boolean;
+	onClick: () => void;
+};
+
+export default function PatternAssemblerButton( { isPrimary, onClick }: Props ) {
```

Comment:
[nit] It would be better to create a new folder for it so we can have a better file structure under the /themes folder.